delfin/task_manager/tasks/resources.py

```diff
@@ -411,13 +411,18 @@ def sync(self):
                  .format(self.storage_id))
         try:
             # collect the filesystems list from driver and database
-            storage_filesystems = self.driver_api.list_filesystems(
+            filesystems, quotas = self.driver_api.list_filesystems(
                 self.context, self.storage_id)
```

Comment:
why not regard quota as a resource? I am curious how it handle if filesystem doesn't have quota

Reply:
Modified quota as resource